src/main/java/com/routeme/api/UserController.java

```diff
@@ -60,6 +60,27 @@ void takeRoute(@RequestBody @Valid EventDTO eventEntry) {
         predictionIOClient.takeRoute(eventEntry.getUserId(), eventEntry.getTargetEntityId());
     }
 
+    @RequestMapping(value = "/like", method = RequestMethod.POST, produces = "application/json")
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://www.codacy.com/app/heshamhamdymassoud/RouteMe-API/file/2647727856/issues/source?bid=3287636&fileBranchId=3323263#l63)